contracts/controller/Avatar.sol

```diff
@@ -121,4 +122,15 @@ contract Avatar is Ownable {
         return true;
     }
 
+    /**
+    * @dev metaData emits an event with a string, should contain the hash of some meta data.
+    * @param _metaData a string representing a hash of the meta data
+    * @return bool which represents a success
+    */
+    function metaData(string memory _metaData) public onlyOwner returns(bool) {
```

Comment:
this is onlyOwner  ..
should we add this api to the controller interface ?

Reply:
Probably yes.
For me it looked like the important thing is to have it on the avatar, as the controller is meant to be upgradable.
But we can decide to support it already in the controller as well.

Reply:
i guess it should also be under certain permission...

Reply:
Also for the time being an org can just put something there on deployment, and only later change it with when we support this feature.

Reply:
you mean via a param to  the DaoCreator ?

Reply:
The permissions issue is one of the reasons I thought to avoid this for the time being. It may be also under the upgrade permission.

Reply:
Yes, regarding the DaoCreator.